webextension/manifest.json

```diff
@@ -28,9 +45,9 @@
     "48": "./skin/sync2cloud.svg",
     "96": "./skin/sync2cloud.svg"
   },
-  "license": "MPL-2.0",
   "manifest_version": 2,
   "permissions": [
-    "management"
+    "management",
+    "storage"
```

Comment:
are you actually using storage?    If it's just for the template, rip this out.  SSAU doesn't actually use storage.

Reply:
Removed